py-polars/tests/unit/operations/test_pivot.py

```diff
@@ -301,7 +356,7 @@ def test_pivot_negative_duration() -> None:
         [pl.Series(name="value", values=range(len(df1) * len(df2)))]
     )
     assert df.pivot(
-        index="delta", columns="root", values="value", aggregate_function=None
+        values="value", index="delta", columns="root", aggregate_function=None
```

Comment:
nitpick, but for ease of review, I'd suggest going through your PR and undoing all irrelevant changes

Reply:
Done.